daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-publish-subscribe.md

```diff
@@ -153,7 +153,7 @@ import sys
 app = flask.Flask(__name__)
 CORS(app)
 
-@app.route('/dsstatus', methods=['POST'])
```

Comment:
Why is this changed? In the above declaration it uses the dsstatus route:

```yml
apiVersion: dapr.io/v1alpha1
kind: Subscription
metadata:
  name: myevent-subscription
spec:
  topic: deathStarStatus
  route: /dsstatus
  pubsubname: pubsub
scopes:
- app1
- app2
```

Reply:
This should not have changed. Fixinf it.